CHANGELOG.md

```diff
@@ -28,7 +28,8 @@ files to the new format.
 - [ENHANCEMENT] The `/agent/api/v1/targets` API will now include discovered labels
   on the target pre-relabeling in a `discovered_labels` field. (@rfratto)
 
-- [ENHANCEMENT] Update to Cortex v1.7.0-rc.0 (@rfratto)
```

Comment:
I removed the Cortex entry after thinking about it and realizing Cortex isn't really directly user-facing, but Loki is (via Promtail)